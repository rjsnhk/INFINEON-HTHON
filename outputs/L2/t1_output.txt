Usage: java Main <model.xml> <queries.txt>
