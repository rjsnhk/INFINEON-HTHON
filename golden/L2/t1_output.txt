Gold captured: 0
Gold captured: 15
Gold captured: 50